crates/compiler/parse/src/normalize.rs

```diff
@@ -835,6 +840,30 @@ fn fold_defs<'a>(
                             arena.alloc(Loc::at_zero(new_final)),
                         );
                     }
+                    ValueDef::Stmt(&Loc {
```

Comment:
Presumably this could be combined with the above clause if args was type `&'a [Loc<Expr<'a>>]` for both of these Expr types.

Reply:
Yep it could.  But that can't happen because of what I said about the use of Collection